src/components/liveRegion.ts

```diff
@@ -0,0 +1,82 @@
+import Zodiac from '../zodiac';
+
+import { ComponentBase } from './componentBase';
+
+/**
+ * Adds a live region, so the slide position can be announced to screen readers.
+ */
+export class LiveRegion extends ComponentBase {
+
+  /**
+   * The live region element.
+   */
+  protected liveRegion: HTMLDivElement;
+
+  /**
+   * {@inheritDoc ComponentBase.mount}
+   */
+  public mount(zodiac: Zodiac): void {
+    super.mount(zodiac);
+
+    if (this.options.enableLiveRegion) {
+      this.createLiveRegion();
+      this.updateLiveRegion();
+    }
+  }
+
+  /**
+   * Creates and adds the live region element to the slider.
+   */
+  protected createLiveRegion() {
+    this.liveRegion = document.createElement('div');
+    this.liveRegion.setAttribute('aria-live', 'polite');
+    this.liveRegion.setAttribute('aria-atomic', 'true');
+    this.liveRegion.classList.add('zodiac-live-region');
+
+    this.zodiac.getSliderElement().appendChild(this.liveRegion);
+  }
+
+  /**
+   * Retrieves the title of the ative item that will be used in the live region.
+   *
+   * The title is expected to be placed in the `data-zodiac-live-region-title`
+   * attribute. This can be on a `zodiac-item` element, or within.
+   *
+   * @returns The title of the active slider item.
+   */
+  protected getLiveRegionTitle(): string {
+    const activeItem = this.zodiac.getSliderElement().querySelector<HTMLElement>('.zodiac-item.active');
+
+    let title = '';
+
+    if (activeItem.dataset.zodiacLiveRegionTitle) {
+      title = activeItem.dataset.zodiacLiveRegionTitle;
+    } else {
+      const element = activeItem.querySelector<HTMLElement>('[data-zodiac-live-region-title]');
+
+      if (element) {
+        title = element.dataset.zodiacLiveRegionTitle;
+      }
+    }
```

Comment:
What's the rationale behind this logic? I'm assuming this is to support the title attribute on nested elements, but just want to be sure. If my assumption is correct, it may be better to just have the `else` condition as the only code path since you'll always get the first result that way if one exists.

May also be more efficient if we can get away with a single `querySelector()` call somehow (e.g., `.zodiac-item.active[data-zodiac-live-region-title], .zodiac-item.active [data-zodiac-live-region-title]`).